src/app/package-json.js

```diff
@@ -168,13 +168,6 @@ const packageJSON = (current, context) => {
     'author': context.author,
     'license': context.licenseName,
 
-    'browserify': {
-      transform: [
-        'browserify-shim',
-        'browserify-versionify'
```

Comment:
Just a note. I added this on a project and found there is a small bug in the build script. 

Line 143 should read: 

`'browserify . -t browserify-shim -t browserify-versionify -s %s -o dist/%s.js'`